src/analyses/custom_bitvector_analysis.cpp

```diff
@@ -208,9 +208,9 @@ std::set<exprt> custom_bitvector_analysist::aliases(
 
     std::set<exprt> result;
 
-    for(const auto &pointer : pointer_set)
-      if(pointer.type().id()==ID_pointer)
-        result.insert(dereference_exprt(pointer));
+    for(const auto &p : pointer_set)
```

Comment:
I'd have preferred renaming the previous pointer definition in order not to end up with undescriptive, one-letter variable names.

Reply:
Thanks for forcing me not to be lazy :-)  - I've gone for "alias", and also renamed "pointer_set" to "alias_set."